bio2zarr/cli.py

```diff
@@ -630,3 +630,71 @@ def vcfpartition(vcfs, verbose, num_partitions, partition_size):
         )
         for region in regions:
             click.echo(f"{region}\t{vcf_path}")
+
+
+@click.command(name="convert")
+@click.argument("ts_path", type=click.Path(exists=True))
+@click.argument("zarr_path", type=click.Path())
+@click.option(
```

Comment:
Let's not bother with ploidy - we only support tskit files that have individuals here. Older files can go through the tskit VCF route, or just add the appropriate individuals.